index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Ayoub Chahir | Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <ul>
+        <li id="logo"><a href="#" class="logo">My Logo</a></li>
+        <li><img src="./img/sandwich.svg" alt="Menu Button" /></li>
```

Comment:
- [ ] @AyoubMs you did well but I think you can still improve your code by using semantic elements according to the project requirements, kindly wrap your menu button tag in `nav` tag  example
 `<nav><img src="./img/sandwich.svg" alt="Menu Button" /></nav>`

Reply:
It gave me a linter error, saying that "nav" not allowed as child of element "ul"